config/install/context.context.compound_member_object.yml

```diff
@@ -17,7 +17,6 @@ conditions:
     parent_reference_field: field_member_of
     model_uri: 'http://vocab.getty.edu/aat/300242735'
     logic: and
-    uri: 'http://purl.org/dc/dcmitype/Collection'
```

Comment:
The PR is currently failing to do a drush site-install, i'm testing whether this is the line that's causing (sorry for misremembered wording) "Config could not be enabled."

Reply:
Hi Rosie - I'm not able to replicate this. Did you run `make local` which contains `docker-compose exec -T drupal with-contenv bash -lc "drush si -y islandora_install_profile_demo --account-pass $(shell cat secrets/live/DRUPAL_DEFAULT_ACCOUNT_PASSWORD)"` or a different function?